compiler/ballerina-lang/src/main/java/org/ballerinalang/model/tree/statements/AssignmentNode.java

```diff
@@ -1,41 +1,43 @@
-/*
-*  Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
-*
-*  WSO2 Inc. licenses this file to you under the Apache License,
-*  Version 2.0 (the "License"); you may not use this file except
-*  in compliance with the License.
-*  You may obtain a copy of the License at
-*
-*    http://www.apache.org/licenses/LICENSE-2.0
-*
-*  Unless required by applicable law or agreed to in writing,
-*  software distributed under the License is distributed on an
-*  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
-*  KIND, either express or implied.  See the License for the
-*  specific language governing permissions and limitations
-*  under the License.
-*/
-package org.ballerinalang.model.tree.statements;
-
-import org.ballerinalang.model.tree.expressions.ExpressionNode;
-import org.ballerinalang.model.tree.expressions.VariableReferenceNode;
-
-/**
- * i = a + b;.
- * p.name, j[0], u = split();.
- *
- * @since 0.94
- */
-public interface AssignmentNode extends StatementNode {
-    ExpressionNode getVariable();
-
-    ExpressionNode getExpression();
-
-    boolean isDeclaredWithVar();
-
-    void setExpression(ExpressionNode expression);
-
-    void setDeclaredWithVar(boolean isDeclaredWithVar);
-
-    void setVariable(VariableReferenceNode variableReferenceNode);
-}
+/*

```

Comment:
Wondering why these are being shown as diffs. When compared with the upstream/master, only the actual diff is in the file

Reply:
If you hide whitespace diff on Github you will see the same diff you find when you compare with master.